frontend/src/hooks/useFormatTableData.tsx

```diff
@@ -75,8 +75,19 @@ export const useFormatTableData = ({
   apiEndPoint: string;
   handleClick?: Function;
 }) => {
+  // const handleCurrentUser = async () => {
```

Comment:
are you going to be implementing this code at some point, also noticed in the useAuthorization hook, or something similar.